src/app/app-routing.module.ts

```diff
@@ -84,7 +76,13 @@ const PORTLETS_IFRAME = [
         children: [
             {
                 component: IframePortletLegacyComponent,
-                path: ':id'
+                path: ':id',
+                children: [
```

Comment:
Now we can do like:

`/c/site-browser/INODE`
`/c/content/INODE`
`/c/vanity-urls/INODE`

And will open the dialog to edit the inode contentlet